docs/git-comparison.md

```diff
@@ -110,13 +111,15 @@ parent.
       <td><code>git clone &lt;source&gt; &lt;destination&gt; [--origin &lt;remote name&gt;]</code></td>
     </tr>
     <tr>
-      <td>Update the local repo with all bookmarks from a remote</td>
+      <!-- TODO: Mention that you might need to do a `jj branch track branch@remote` -->
```

Comment:
I’m not sure where this TODO is from, but it seems outdated.

Reply:
I just added it :) It's obviously not clear enough, but honestly it's unclear to me what to do here.

The problem is that if you `jj git fetch` from a remote, you'll get all the bookmarks, but you won't see them in `jj log` until you `jj bookmark track` something. I think new people are likely to be confused by that, but I'm not sure how to help. (I already inserted an explanation in a few places, it's not 100% sure this has to be one of them, but it felt relevant).

Reply:
I'll update the TODO in a moment.

Reply:
To be clear the outdated part was just that it’s talking about `jj branch` :)

Reply:
Ah, great point, thanks!